src/Assessor/Filter/AssessorRequestFilters.php

```diff
@@ -55,6 +78,58 @@ public function apply(QueryBuilder $qb, string $alias): void
             ;
         }
 
+        if ($this->getLastName()) {
+            $qb
+                ->andWhere("$alias.lastName = :lastName")
+                ->setParameter('lastName', $this->getLastName())
+            ;
+        }
+
+        if ($this->getCity()) {
+            if (is_numeric($this->getCity())) {
+                $qb
+                    ->andWhere("$alias.assessorPostalCode LIKE :assessorPostalCode")
+                    ->setParameter('assessorPostalCode', $this->getCity().'%')
+                ;
+            } else {
+                $qb
+                    ->andWhere("LOWER($alias.assessorCity) LIKE :assessorCity")
+                    ->setParameter('assessorCity', '%'.strtolower($this->getCity()).'%')
+                ;
+            }
+        }
+
+        if ($this->getCountry()) {
+            $qb
+                ->andWhere("$alias.assessorCountry = :assessorCountry")
+                ->setParameter('assessorCountry', $this->getCountry())
+            ;
+        }
+
+        if ($this->getVotePlace()) {
+            if ($this->isStatusUnprocessed()) {
+                $qb->innerJoin("$alias.votePlaceWishes", 'vp');
+            } else {
+                $qb->innerJoin("$alias.votePlace", 'vp');
+            }
+
+            if (preg_match('/([0-9]{5}|2[A-Z][0-9]{3})_[0-9]{4}/', $this->getVotePlace())) {
```

Comment:
What do you think to use a const variable to put the `/([0-9]{5}|2[A-Z][0-9]{3})_[0-9]{4}/` regex in the `AssessorFilters` class ?